pbs2slurm.py

```diff
@@ -231,9 +232,7 @@ def fix_resource_list(pbs_directives):
     if l_m is not None:
         def _repl(m):
             resources = m.group(1)
-            if not "walltime" in resources:
-                return ""
-            else:
+            if "walltime" in resources:
```

Comment:
The problem with this is that a resource list that includes "walltime" will not be parsed for other entries. For example
```
#PBS -l nodes=1:htown:gige:ppn=8,walltime=60:00:00
```
will translate only to
```
#SBATCH --time=60:00:00
```